subsys/bluetooth/mesh/Kconfig

```diff
@@ -299,6 +299,15 @@ config BT_MESH_ADV_EXT
 	  Extended advertising is faster and uses less memory than legacy
 	  advertising, but isn't supported by all controllers.
 
+config BT_MESH_ADV_EXT_MULTI
+	bool "Multi Extended advertising sets"
+	depends on BT_CTLR_ADV_EXT || !BT_CTLR
+	depends on BT_EXT_ADV
+	help
+	  Use multi extended advertising commands for operating the advertiser.
+	  Multi extended advertising is faster than legacy or single extended
+	  advertising, but isn't supported by all controllers.
```

Comment:
> but isn't supported by all controllers.

How does that piece of information help the reader ?
I assume the above `depends on` settings ensures that this setting can only be enabled if the currently selected controller supports `BT_MESH_ADV_EXT_MULTI` or do we need some additional Kconfig settings so that proper dependency chain can be ensured.